lib/liquid/combinators/general.ex

```diff
@@ -138,6 +139,8 @@ defmodule Liquid.Combinators.General do
   Comparison operators:
   == != > < >= <=
   """
+  @type comparison_operators :: :== | :!= | :> | :< | :>= | :<= | :contains
```

Comment:
all type declaration must be grouped before functions:
```
@moduledoc
@behaviour
use
import
alias
require
defstruct
@type                  # <-------------- here
@module_attribute
@callback
@macrocallback
@optional_callbacks


functions...
```
See it here: [Style Guide](https://github.com/christopheradams/elixir_style_guide#module-attribute-ordering)